mct/m_SPMDutils.F90

```diff
@@ -24,6 +24,16 @@
 !
 ! !INTERFACE:
 
+! Code added as a work around for poor performance or incorrect functionality
+! in MPI libraries
+!
+#ifndef _USE_MPI_RSEND
```

Comment:
Yes please don't clean up the white space.  I'll do that here with a separate branch that will bring it in line with ACME/ESMCI.